### DISCUSSION TITLE
Pedido de consigna TPA4

### DISCUSSION THREAD

**Discussion by mbeorlegui** [dds-jv/foro#382] [Consultas generales] [answered]
Buenas Gastón, necesitamos que nos envíen la consigna del TP a la brevedad porque se entrega en poco tiempo.
No te hablé por el canal de consultas urgentes de Discord porque me aparece como inhabilitado.

Saludos y buen finde.

**Comment by gastonprieto** [comment, marked as answer]
buenas, ya esta enviada
